cmd/server/server.go

```diff
@@ -390,8 +390,12 @@ func (s *server) createDatastore(ctx context.Context, cfg *config.ControlPlaneSp
 		options := []mongodb.Option{
 			mongodb.WithLogger(logger),
 		}
-		return mongodb.NewMongoDB(ctx, mdConfig.URL, mdConfig.Database, options...)
-
+		if mdConfig.UsernameFile != "" || mdConfig.PasswordFile != "" {
+			options = append(options, mongodb.WithAuthenticationFile(mdConfig.UsernameFile, mdConfig.PasswordFile))
+		}
+		return mongodb.NewMongoDB(ctx, mdConfig.URL,
+			mdConfig.Database,
+			options...)
```

Comment:
nit: Just want to be unified with our coding conventions.

```
mongodb.NewMongoDB(
   ctx,
   mdConfig.URL,
   ...,
)
```

Reply:
@nghialv 
Thanks for your review.
I changed it to match the coding conventions. Please check it again when you have time :+1: :+1: